localstack/services/events/provider.py

```diff
@@ -337,7 +374,7 @@ def filter_event(event_pattern_filter: Dict[str, Any], event: Dict[str, Any]):
                         if not handle_prefix_filtering(value.get(key_a), value_a):
                             return False
 
-            # 2. check if the pattern is a list and event values are not contained in it
+            # 2. check if the pattern is a list and event values are not contained in itEventsApi
```

Comment:
nit: typo

Reply:
I didn't see this comment, my bad. I'll correct it in my next PR.